runtime/lib/backend/lightning/lightning_kokkos/LightningKokkosSimulator.cpp

```diff
@@ -455,7 +459,8 @@ auto LightningKokkosSimulator::Measure(QubitIdType wire, std::optional<int32_t>
     SetDeviceShots(device_shots);
 
     // It represents the measured result, true for 1, false for 0
-    bool mres = Lightning::simulateDraw(probs, postselect);
+    bool has_seed = (this->seed != "");
```

Comment:
Maybe it would make more sense for `has_seed` to be a device class variable that can be set from the context, where the default argument of the `ExecutionContext` was set as empty string, like the seed and the prng pointer,  instead of making this comparison here? Otherwise a developer would have to remember what empty string means at this point.

Reply:
... or a device `has_seed` method